R/misc.R

```diff
@@ -167,6 +165,15 @@ filter_Y <- function(Y, n_nonmiss) {
   return(list(Y = Y, rm_rows = rm_rows))
 }
 
+filter_data_driven_priors <- function(data_driven_prior_matrices, prior_weights_min){
```

Comment:
Do we have a step to also include only the context that we specify? Or are we using a different function for that?

Reply:
Sorry this would need to be removed and it was not used anywhere in the pipeline, I missed cleaning up this one.